### ISSUE TITLE
ENH: Auto look for `_polished.yml`

### ISSUE THREAD

**Issue by muschellij2** [Tychobra/polished#164]
ENH - close if needed.  

I wonder if some code could be useful that if you pass in `NULL` or have `NULL` as the default for `global_sessions_config` arguments, then it could look for a `_polished.yml` file, similar to `pkgdown` and `bookdown`.
```r
  polished_yaml_file = "_polished.yml"
  if (file.exists(polished_yaml_file)) {
    polished_config = config::get(file = polished_yaml_file)
  }
```

**Comment by merlinoa** [comment]
Yes I think this could be useful.  I'll give it some more thought over the next couple days.
